docs/export.md

```diff
@@ -120,11 +124,13 @@ The export tool is going to look for the following at a minimum:
 
 Now, we could export just with these artifacts an would get a .model file. It's going to look at the signature file and see that you're using `"input_type": "image/jpeg"`, and assume that you want the default vision service, so it will include `mxnet_vision_service.py` for you. It will also generate the manifest. Let's try it out.
 
-We're going to tell it our model's name is `squeezenet_v1.1` with the `--model-name` argument. The name works like a prefix, so it will assume that you've named the symbol file and the params file according to the pattern _name_-symbol.json and _name_-0000.params. The "0000" can be another checkpoint if that's what you have. Then we're giving it the `--model-path` to the model's assets, which are in the current working directory, so we'll use `.` for the path.
+We're going to tell it our model's name is `squeezenet_v1.1` with the `--model-name` argument. The name works like a prefix, so it will assume that you've named the symbol file and the params file according to the pattern _name_-symbol.json and _name_-0000.params. The "0000" can be another checkpoint if that's what you have. Then we're giving it the `--model-path` to the model's assets, which are in the current working directory, so we'll use `.` for the path. We also provide an optional argument `from-type` to indicate which model type we want to pack into model archive. Currently, MXNet model and ONNX model can be exported.
```

Comment:
This example is specifically for an MXNet model, not ONNX.
To make it more clear, why don't we add a new section `## Export ONNX Model Example`, that will specify separately the details of exporting an ONNX model?

Reply:
Let's break this onnx example out separately so as not to confuse. Also, since you're going to detect the model type automatically, we really don't want to include that extra param in the basic usage example.

Add a new section after line 134.
## ONNX Export Example

MMS supports the ONNX format and using it ONNX models is very simple. If the model has the `.onnx` extension it will be detected and the managed accordingly. However, if you need to specify a model as the ONNX format, you can use the optional `from-type` argument.

```
mxnet-model-export --model-name squeezenet_v1.1 --model-path . --from-type onnx
```

Reply:
Looks good!
Small nits:
- "MMS supports the ONNX format and using it ONNX models is very simple." -> "MMS supports the ONNX format and using ONNX models is very simple."
- "you can use the optional from-type argument." -> "you can use the optional model-format argument."